.github/workflows/check-feds-files.yaml

```diff
@@ -0,0 +1,25 @@
+name: Feds File Watch
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened]
```

Comment:
Might get a little spammy if you always send a message on edited... maybe on open and merge, or when the ready for stage label is applied?

Reply:
I changed it to run only on labeled and on open(without checking the label to avoid the back and forth in case I get the label name wrong).  That way we get a run if labels are added when the PR is raised and one guaranteed run when ready for stage is applied.